docker-compose.yml

```diff
@@ -15,13 +15,14 @@ services:
     depends_on:
       - display
       - video
+      - agd
     entrypoint: []
     working_dir: /app
     volumes:
       - ./docker/videos:/app/tests/e2e/videos
       - ./docker/screenshots:/app/tests/e2e/screenshots
     command: >
-      bash -c 'echo -n "======> local noVNC URL: http://localhost:8080/vnc.html?autoconnect=true " && pnpm wait-on http://display:8080 && echo -n "======> remote noVNC URL: " && curl -s ngrok:4040/api/tunnels | jq -r .tunnels[0].public_url && nginx && pnpm test:e2e:anvil'
```

Comment:
what is anvil used for? Is this change going to disable metamask tests?

Reply:
yes the file is currently only running our keplr tests. anvil is a tool used in the metamask tests

Reply:
created the issue